client/src/components/Sub_RelatedItems/Sub_SingleProd/ComparisonModal.jsx

```diff
@@ -63,27 +50,41 @@ export default function ComparisonModal({
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
     >
-      <table className="comparison-modal-content">
-        <tbody className="comparison-modal-header">
-          <tr>
-            <th>{thisProduct.name}</th>
-            <th>vs.</th>
-            <th>{parentProduct.name}</th>
-          </tr>
-          <tr className="overall-rating">
-            <td>{thisAvgRating}</td>
-            <td>Overall Rating</td>
-            <td>{parentAvgRating}</td>
-          </tr>
-          {combinedStats.map((nestedStats, i) => (
-            <tr key={i} className="additional-rating">
-              <td>{nestedStats[0]}</td>
-              <td>{nestedStats[1]}</td>
-              <td>{nestedStats[2]}</td>
+      <div className="comparison-modal-content">
+        <table>
+          <tbody className="comparison-modal-body">
+            <tr>
+              <td>{null}</td>
+              <td>{null}</td>
+              <td>
+                <img
+                  className="remove-x"
+                  src="../../../../assets/remove2.png"
+                  alt="close this modal"
+                  onClick={() => setShowComparisonModal(false)}
+                />
+              </td>
```

Comment:
I see that you have a button ish image with remove2.png, what are the other 2 td elements are for? asking just because I would want to know

Reply:
The two `null` entries allowed me to position the X button more accurately (to the far right side).